### DISCUSSION TITLE
Can't access online content

### DISCUSSION THREAD

**Discussion by Gabixel** [ppy/osu#15343] [Q&A] [answered]
I don't know what happened, but after downgrading and re-upgrading (to check for possible regression) I can't access/view anything related to the API/web (e.g. login, wiki, news).
Here are the logs:
[`network.log`](https://github.com/ppy/osu/files/7433004/network.log)
[`runtime.log`](https://github.com/ppy/osu/files/7433005/runtime.log)
[`updater.log`](https://github.com/ppy/osu/files/7433006/updater.log)

I have no idea if reinstalling the app is enough, but I'll wait if I have to help analyze this further.

**Comment by peppy** [comment, marked as answer]
Log out then log in.

**Reply by Gabixel** [reply]
That's another problem: I wasn't logged in.
In the end, I even tried reinstalling the app. For a moment, I was able to load some content, but after that, it started producing these errors again.
I checked if there were problems with my connection ~but that doesn't seem to be the case~ and right now, in my area, there are problems with various services (such as Discord and online gaming). So the errors may be caused by that.
